### PR TITLE
Support kwargs for send_sms

### PR DESCRIPTION
**Pull request by anonaka**
Skip failed tests

### CODE REVIEW

File: tests/test_cloudpy.py [hunk 1]
```diff
@@ -124,27 +141,35 @@ def test_get_sms_delivery_status_fail01(self):
             msgid="dabd8e76-390f-421c-87b5-57f31339d0c5")
         self.assertEqual(xoxzo_res.errors, 404)
 
+    @unittest.skip("This test fails due to rate limiting")
     def test_get_sms_list_success01(self):
+        time.sleep(5)
```
Comment (za) [inline tests/test_cloudpy.py:148]: @anonaka Even after we add sleep here, does the test still fail due to rate limiting?

File: setup.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="xoxzo.cloudpy",
-    version="0.4.1",
+    version="0.5.0",
```
Comment (za) [inline setup.py:5]: @anonaka Do you think is a good time now to initiate `CHANGES.txt` to track the changes? #minor

File: tests/test_cloudpy.py [hunk 3]
```diff
@@ -108,6 +109,22 @@ def test_send_sms_success01(self):
         self.assertTrue('msgid' in xoxzo_res.message)
         self.assertEqual(xoxzo_res.messages,[])
 
+    def test_send_sms_kwargs_success01(self):
+        xoxzo_res = self.xc.send_sms(
+            "Hello from Xoxzo",
+            self.test_recipient,
+            self.test_sender,
+            foo="bar")
```
Comment (arthur-s) [inline tests/test_cloudpy.py]: there's no test for foo="bar". How to make sure, that kwargs works? Maybe you can pass any real kwarg? I'm not familiar, but I think you can pass a link tracking, or KDDI params. @anonaka .

Comment (anonaka) [reply tests/test_cloudpy.py]: Test is updated.

File: tests/test_cloudpy.py (hunk 1 above)
Comment (anonaka) [reply tests/test_cloudpy.py:148]: Right. This API is limited up to three times in an hour.

File: setup.py (hunk 2 above)
Comment (anonaka) [reply setup.py:5]: I have checked the packaging guide but could not find the description about the file "CHANGES.txt". 
https://packaging.python.org
I think it is not necessary at this moment.

**Review by anonaka — COMMENTED** [review]
I have addressed all the comments.

**Pull request merged**